test/pdffonts.test.js

```diff
@@ -19,165 +19,165 @@ describe('pdffonts', () => {
 
     it('returns an array of font objects', () => {
       return PDFFonts.fonts(EMBEDDED_FONTS_PATH)
-      .then((fonts) => {
-        expect(fonts).to.eql([{
-          name: 'LDJWDV+DejaVuSerif-Bold',
-          type: 'CID TrueType',
-          encoding: 'Identity-H',
-          embedded: true,
-          subset: true,
-          unicode: true,
-          object: {
-            number: 8,
-            generation: 0
-          }
-        }]);
-      });
+        .then((fonts) => {
```

Comment:
more of a nit but we could rewrite some of these chains to be async/await for readability in the future

Reply:
good idea!